app.yaml

```diff
@@ -0,0 +1,15 @@
+runtime: python37
+
+handlers:
```

Comment:
Let's add `secure: always` to all of the handlers, and explicitly create a handler for `/api` so that we can also apply `secure: all` to it. (Also I prefer less magic default.)

Reply:
This bit is why I couldn't quick fix and merge the PR, see the commit message in https://github.com/foolip/ecosystem-infra-rotation/commit/a51a17e181ebea4989b4c688e1800af635750b95 for why I suspect this might be take some tinkering.